ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/constants/RdfConstants.java

```diff
@@ -32,4 +32,7 @@ private RdfConstants() {
 	public static final String TREE_LESSER_THAN_OR_EQUAL_TO_RELATION = TREE + "LessThanOrEqualToRelation";
 
 	public static final String GENERIC_TREE_RELATION = TREE + "Relation";
+	public static final String NEXT_PAGE_RELATION = TREE + "Next";
+	public static final String PREVIOUS_PAGE_RELATION = TREE + "Previous";
+	public static final String PAGE_NUMBER = "pageNumber";
```

Comment:
This doesn't seem to be right. Based on what I see here, there should be a "https://w3id.org/tree#Next" and "https://w3id.org/tree#Previous" Relationship. 
But since this isn't spec compliant, I would just use Generic relations for this